### ISSUE TITLE
[Feature]: UI Mode - Set number of workers in ui mode interface

### ISSUE THREAD

**Issue by Kumasta** [microsoft/playwright#30304]
### 🚀 Feature Request

In the UI mode window - add a worker count and have the ability to set number of workers when running tests in UI mode. 

### Example

![image](https://github.com/microsoft/playwright/assets/94964514/06477adb-aafb-4fbd-bc1d-3aaff63bc870)

Add a worker count that can be adjusted in ui mode.

### Motivation

I occasionally want to run tests with just one worker to test stability of tests one by one as well as some tests are run in a single context back to back. 

I would need to close ui mode and add the worker flag to the command prompt to update this.

**Comment by HamzaMiz** [comment]
+1

**Comment by ssloanRIIS** [comment]
I support this.

What I'd also like to see, is workers based off of the project. I want to run most of my tests locally in parallel, but some of my tests _**NEED**_ to be ran in serial. I would imagine that I could configure this per project, but it seems that I'm limited to only setting the amount of workers in the `PlaywrightTestConfig` rather than the project.

```
export default defineConfig({
  testDir: "./tests",
  fullyParallel: true,
  workers: 2,
  retries: 0,
  reporter: [["html"], ["list"]],

  use: {
    baseURL: process.env.BASE_URL || "http://localhost:3000",
    trace: "on-first-retry",
  },

  projects: [
    {
      name: "extra_secret_tests",
      fullyParallel: false, //<------- Disables *IN THE TEST FILE* but not in the suite...
      testDir: "./tests/extra_secret_tests",
      workers: 1, //<------ Why doesn't this work!!
      use: {
        ...devices["Desktop Chrome"],
        viewport: { width: 1920, height: 1080 },
        storageState: STORAGE_STATE,
      },
    },
  ],
});
```

**Comment by marliz97** [comment]
+1

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you have additional information not present in this issue that you think will help prioritizing it, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
